fixtures/status_request.json

```diff
@@ -0,0 +1,4 @@
+{
+   "game_name":"game 1",
+   "player_name":"hank the frog"
+}
```

Comment:
Do we need to pass anything in the body if we are passing the table guid and player guid in the headers?